src/main/java/org/folio/circulation/domain/UpdateRequestService.java

```diff
@@ -46,6 +49,16 @@ public UpdateRequestService(RequestRepository requestRepository,
         .thenApply(r -> r.next(requestNoticeSender::sendNoticeOnRequestUpdated))));
   }
 
+  private Result<Request> refuseWhenPatronCommentChanged(
+    Request updated, Result<Request> originalRequestResult) {
+
+    return originalRequestResult.failWhen(
+      original -> succeeded(!Objects.equals(updated.getPatronComments(), original.getPatronComments())),
+      original -> singleValidationError("Unable to update patronComments for request",
```

Comment:
This message might be presented to the user. What do you think to the language below?

```suggestion
      original -> singleValidationError("Patron comments are not allowed to change",
```

Reply:
Works for me, changed.